py_ecc/bls/hash.py

```diff
@@ -15,3 +18,39 @@ def hash_eth2(data: Union[bytes, bytearray]) -> Hash32:
     a future Ethereum 2.0 deployment phase.
     """
     return Hash32(hashlib.sha256(data).digest())
+
+
+def hkdf_extract(salt: Union[bytes, bytearray], ikm: Union[bytes, bytearray]) -> bytes:
+    """
+    HKDF-Expand
+
+    https://tools.ietf.org/html/rfc5869
+    """
+    return hmac.new(salt, ikm, hashlib.sha256).digest()
+
+
+def hkdf_expand(prk: Union[bytes, bytearray], info: Union[bytes, bytearray], length: int) -> bytes:
+    """
+    HKDF-Expand
+
+    https://tools.ietf.org/html/rfc5869
+    """
+    # n = cieling(length / HASH_LENGTH_BYTES)
+    n = length // HASH_LENGTH_BYTES
+    if n * HASH_LENGTH_BYTES < length:
```

Comment:
I think you can simplify this to:

```python
n = length // HASH_LENGTH_BYTES + length % 2
```

I think it might be a little more intuitive to read.

Reply:
Yep, I like that, thanks :)

Reply:
Actually on second though that wouldn't work. e.g.
`length = 34`, `HASH_LENGTH_BYTES = 32`
=> `n = 34 // 32 + 34 % 2 = 1`
But `n` should be 2.

Reply:
:man_facepalming: